recipes/yt/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py build --compiler=mingw32 install
```

Comment:
When using `setuptool`, we add the following additional arguments after `install`.

    --single-version-externally-managed --record=record.txt

Could you please add these?